cinder/README.md

```diff
@@ -25,15 +25,18 @@ The Dell EMC Cinder container image contains following RPM packages:
 
 The formal Dell EMC container image is published to [Red Hat Container Catalog](https://access.redhat.com/containers/)
 
+Red Hat OpenStack Platform supports remote registry and local registry for overcloud deployment. In this document, we only introduce local registry.
```

Comment:
we only introduce local registry?

Reply:
Yes, we only use local registry in our deployment.
There are very detail instructions for remote registry in Red Hat document, so we don't need to cover it again.

Reply:
Should we mention this to the user?

Reply:
We assume that they have enough knowledge to deploy osp, otherwise there are lots of things we need to mention in this document.